specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -1459,255 +1618,2049 @@
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "SnapshotPolicies_Delete": {
+            "$ref": "examples/SnapshotPolicies_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/snapshotPolicies/{snapshotPolicyName}/listVolumes": {
+      "get": {
+        "tags": [
+          "Snapshot Policy List Volumes"
+        ],
+        "summary": "Get volumes associated with snapshot policy",
+        "description": "Get volumes associated with snapshot policy",
+        "operationId": "SnapshotPolicies_ListVolumes",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/SnapshotPolicyName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/snapshotPolicyVolumeList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "SnapshotPolicies_ListVolumes": {
+            "$ref": "examples/SnapshotPolicies_ListVolumes.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/accountBackups": {
+      "get": {
+        "tags": [
+          "Backups"
+        ],
+        "summary": "List Backups for a Netapp Account",
+        "description": "List all Backups for a Netapp Account",
+        "operationId": "AccountBackups_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/backupsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "AccountBackups_List": {
+            "$ref": "examples/Backups_Account_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/accountBackups/{backupName}": {
```

Comment:
how are backups created/updated? I dont see a PUT?

Reply:
Backups are done via /backup , this GETs backups for given account